packages/apps/shopify-app-remix/src/server/authenticate/admin/billing/authenticate.admin.billing.doc.ts

```diff
@@ -3,7 +3,8 @@ import {ReferenceEntityTemplateSchema} from '@shopify/generate-docs';
 const data: ReferenceEntityTemplateSchema = {
   name: 'Billing',
   description:
-    'Contains function used to bill merchants for your app.\n\nThis object is returned on authenticated Admin requests.',
+    'Contains function used to bill merchants for your app with the Billing API.\n\nThis object is returned on authenticated Admin requests.' +
+    '\n\n> Note: [Managed App Pricing](/docs/apps/launch/billing/managed-pricing) is now available. Define your app’s pricing plans directly in the Shopify Partner Dashboard, without needing to use the Billing API.',
```

Comment:
I am wondering if we want to add a full section on billing, and have more details on setting up both API based billing and managed pricing. Similar to how [we sections for both API and toml webhooks](https://shopify.dev/docs/api/shopify-app-remix/v3/guide-webhooks)?